x/supply/types/query.pb.gw.go

```diff
@@ -72,22 +70,19 @@ func local_request_Query_LiquidSupply_0(ctx context.Context, marshaler runtime.M
 // RegisterQueryHandlerServer registers the http handlers for service Query to "mux".
 // UnaryRPC     :call QueryServer directly.
 // StreamingRPC :currently unsupported pending https://github.com/grpc/grpc-go/issues/906.
-// Note that using this registration option will cause many gRPC library features to stop working. Consider using RegisterQueryHandlerFromEndpoint instead.
+// Note that using this registration option will cause many gRPC library features (such as grpc.SendHeader, etc) to stop working. Consider using RegisterQueryHandlerFromEndpoint instead.
 func RegisterQueryHandlerServer(ctx context.Context, mux *runtime.ServeMux, server QueryServer) error {
 
 	mux.Handle("GET", pattern_Query_TotalSupply_0, func(w http.ResponseWriter, req *http.Request, pathParams map[string]string) {
 		ctx, cancel := context.WithCancel(req.Context())
 		defer cancel()
-		var stream runtime.ServerTransportStream
-		ctx = grpc.NewContextWithServerTransportStream(ctx, &stream)
```

Comment:
these aren't needed?

Reply:
@devashishdxt

Reply:
These files are auto-generated by protobuf compiler.

Reply:
yes, I installed the version specified in cosmos repo, maybe there's upgrades.

Reply:
is supply module's GRPC query tested in integration tests?
I understand this is auto-generated, but to be sure this doesn't produce invalid code / it'd crash (e.g. due to mismatch between the protobuf generator and the Go dependency library)

Reply:
I think supply module is not tested in integration test, we can do that in another issue.
Test cli query should be good, cli query calls the grpc api.

Reply:
I just did a manual simple test on cli, it do works.
```
$ pystarport serve --config examples/devnet.yaml
```
```
$ chain-maind q supply liquid
supply:
- amount: "40000000000"
  denom: basecro
```
```
$ chain-maind q supply total
supply:
- amount: "62000003831"
  denom: basecro
```